### DISCUSSION TITLE
How to ensure the approval time of PR?

### DISCUSSION THREAD

**Discussion by fbs2016** [microsoft/winget-pkgs#58054] [General] [answered]
I'm new comer of winget. My boss want to deliver our product to winget.
I have some question about PR, could someone can help answer it? Thanks
How long it takes to finish the PR assume that my pkg is valid?
For example, I submit the PR on winget-pkgs, the pipeline has passed and just wait for approval of moderator.

How to ensure the accept time so that it doesn't block the deliver in next several days of plan .

**Comment by OfficialEsco** [comment, marked as answer]
We Moderators are watching the PRs 24/7, so unless your Package is getting hit by any of the red labels it will be approved and merged within hours unless everyone is busy that specific day.
In the future this will also be fully automated once wingetbot learns to control its power.
